taplo.toml

```diff
@@ -0,0 +1,26 @@
+[formatting]
+# Align consecutive entries vertically.
+align_entries = false
+# Append trailing commas for multi-line arrays.
+array_trailing_comma = true
+# Expand arrays to multiple lines that exceed the maximum column width.
+array_auto_expand = true
+# Collapse arrays that don't exceed the maximum column width and don't contain comments.
+array_auto_collapse = false
+# Omit white space padding from single-line arrays
+compact_arrays = true
```

Comment:
Personally, I like the space between the `[` and `]` in arrays because it is easier on my eyes, if you don't have a preference my suggestion is:

```suggestion
# Include white space padding in single-line arrays for better readability.
compact_arrays = false
```